app/locale/en.coffee

```diff
@@ -1060,8 +1060,28 @@
     robin_title: "Senior Product Manager"
     nolan_title: "Sales Manager"
     david_title: "Marketing Lead"
-    titles_csm: "Customer Success Manager"
```

Comment:
Test failing due to omission of this line and line beneath.

To remove a line it must be removed from every translation file as well.